Doc/conf.py

```diff
@@ -286,7 +286,7 @@
     print("It may be removed in the future\n")
 
 # Short title used e.g. for <title> HTML tags.
-html_short_title = f'{release} Documentation'
+html_short_title = f'Python {version} Documentation'
```

Comment:
I agree with Bénédikt -- we should include full version info.
```suggestion
html_short_title = f'Python {release} Documentation'
```